docs/src/docs/asciidoc/user/usermanaged.adoc

```diff
@@ -56,88 +56,35 @@ include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[
-to copy the keys and values.
+* <<serializers-copiers.adoc#,Serializers and copiers>>
+* <<eviction-advisor.adoc#,Eviction advisor>>
 
-=== Cache serializing copier and serializer example
+Simply use the methods from `UserManagedCacheBuilder` which are equivalent to the ones from `CacheConfigurationBuilder`.
 
-[source,java,indent=0]
-----
-include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[tag=userManagedSerializingCache]
-----
-<1> Configure a key serializer.
-<2> Configure a value serializer.
-<3> Configure a `SerializingCopier` is the value copier.
-<4> Configure a heap tier.
-<5> Configure a off-heap tier.
+Below we will describe some more advanced setup where there is need to maintain a service instance in order to have working user managed cache.
 
-In this case, the heap tier is storing keys by reference since no key copier has been configured, and it is storing
-values by value since a serializing copier has been configured. Such copier is going to use the configured
-serializer to perform the necessary copies.
-The off-heap tier will use both the key serializer as well as the value one to make its copies.
+=== Example with disk persistent and lifecycle
 
-=== Write-Through Cache example
+If you want to use disk persistent cache, you will need to create and lifecycle the persistence service.
 
 [source,java,indent=0]
 ----
-include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[tag=userManagedLoaderWriterCache]
+include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[tag=persistentUserManagedCache]
 ----
-<1> If you wish to use a cache in read-through/write-through caching pattern, you'll have to implement
-    `CacheLoaderWriter` and register it.
-
-=== Cache with Eviction Advisor example
+<1> Create the persistence service to be used by the cache for storing data on disk
+<2> Pass the persistence service to the builder next to an id for the cache - note that this will make the builder produce a more specific type: `PersistentUserManagedCache`
+<3> As usual, indicate here if the data should outlive the cache
+<4> Closing the cache will not delete the data it saved on disk when marked as persistent.
+<5> To delete the data, after closing the cache, destroy has to be explicitly invoked.
+<6> It is also your responsibility to stop the persistence service once you are done with the cache.
 
-[source,java,indent=0]
-----
-include::{sourcedir}/impl/src/test/java/org/ehcache/docs/UserManagedCaches.java[tag=userManagedEvictionAdvisorCache]
-----
-<1> If you want to hint the eviction algorithm to advise against the eviction of some mappings, you have to
-    configure an instance of `EvictionAdvisor`.
+=== Example with cache event listeners
 
-=== Cache with Listeners example
+If you want to use cache event listeners, you need to provide the executor services with which the events are to be fired.
```

Comment:
1) should executor service be plural here?
2) this fragment: "with which the events are to be fired" doesn't feel natural
3) nothing about the actual content here